Help/FAQ.rst

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
whaaa.  GitHub is being silly.  This comment was about a completely different blank line, which has been removed.
So this comment is no longer relevant.